session-3-backend-api/README.md

```diff
@@ -198,17 +196,15 @@ Notice that Postman automatically sets one property in the header.
 Content-Type: application/json
 ```
 
-`Content-Type` in header tells the server what is inside our `body`, then server knows how to interpret/understand the body string.
-In this case, the server will interpret the body string as a JSON/JavaScript object. 
+`Content-Type` in header tells the server what is inside our `body`. That way, the server knows how to interpret/understand the body string. In this case, the server will interpret the body string as a JSON/JavaScript object. 
```

Comment:
The body is not necessarily interpreted as a JavaScript string. I’d just say “the body.”

The `Content-Type` header…

… as JSON, or a JavaScript object. (JSON is a mass noun.)